background.js

```diff
@@ -202,3 +202,25 @@ chrome.storage.local.get('version', prefs => {
 		}
 	}
 });
+
+chrome.runtime.onInstalled.addListener(({reason, previousVersion, id} = {}) => {
```

Comment:
note that `chrome.runtime.onInstalled.addListener` callback is not getting fired after extension gets enabled. Not really important, just a note. For instance if a tab gets opened when extension is disabled, and user enables the extension afterwards, it is not going to style those pages.

btw, we want to execute this function on startup anyway, is there any reason why this thing is inside the callback?